tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/CodeownersHelper.cs

```diff
@@ -0,0 +1,512 @@
+using System.Text.RegularExpressions;
+using Azure.Sdk.Tools.Cli.Models.Responses;
+using Azure.Sdk.Tools.CodeownersUtils.Parsing;
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    public interface ICodeownersHelper
+    {
+        CodeownersEntry? FindMatchingEntries(IList<CodeownersEntry> entries, string path = null, string serviceLabel = null);
+        (CodeownersEntry entry, bool existed) UpdateOrCreateCodeownersEntry(
+            CodeownersEntry? existingEntry,
+            string path,
+            string serviceLabel,
+            List<string> serviceOwners,
+            List<string> sourceOwners,
+            bool isAdding);
+        string NormalizePath(string path);
+        CodeownersEntry findAlphabeticalInsertionPoint(List<CodeownersEntry> codeownersEntries, CodeownersEntry codeownersEntry);
```

Comment:
```suggestion
        CodeownersEntry FindAlphabeticalInsertionPoint(List<CodeownersEntry> codeownersEntries, CodeownersEntry codeownersEntry);
```

Public members are usually PascalCase